subprojects/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r22/ClientShutdownCrossVersionSpec.groovy

```diff
@@ -36,6 +37,10 @@ class ClientShutdownCrossVersionSpec extends ToolingApiSpecification {
         toolingApi.close()
     }
 
+    private List<String> getBuildJvmArguments() {
```

Comment:
I'd put this private method at the end of the class.

Reply:
done.